### DISCUSSION TITLE
CVE 2023-50387 in airflow image 2.9.2

### DISCUSSION THREAD

**Discussion by gsingh935** [apache/airflow#40906] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### If "Other Airflow 2 version" selected, which one?

2.9.2

### What happened?

CVE 2023-50387 in airflow image 2.9.2     systemd 252.22-1~deb12u1 12007416

### How to reproduce

Container scanning of image 2.9.2

### Operating System

ubuntu

### Versions of Apache Airflow Providers

None

### Deployment

Official Apache Airflow Helm Chart

### Deployment details

AKS

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
See https://airflow.apache.org/docs/docker-stack/index.html#what-should-i-do-if-my-security-scan-shows-critical-and-high-vulnerabilities-in-the-image - it explains what you can do to get rid of it.
